content/en/blog/_posts/2020-01-14-kubernetes-bug-bounty-announcement.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog
+title: "Announcing the Kubernetes bug bounty program"
+date: 2020-01-14T09:00:00-08:00
```

Comment:
This date is in the past; that's OK, right?

Reply:
Thanks Tim, in the interest of time we're going to leave as is.